internal/postprocessor/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// Vet runs linters on all .go files recursively from the given directory.
+func Vet(dir string) error {
+	log.Println("vetting generated code")
+	c := exec.Command("gofmt", "-s", "-d", "-w", "-l", ".")
```

Comment:
I think at one time you had copied in our modified exec package to the post-processor. Let's move that code back in. It helps provide better debug logging if needed.